src/main/java/StringAddCal.java

```diff
@@ -0,0 +1,52 @@
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCal {
+    public static int splitAndSum(String input) throws RuntimeException {
+        if (nullOrEmpty(input) == false) {
+         return 0;
+        }
+
+        String[] split = customDelimeter(input);
+
+        return calAllSum(split);
+    }
+
+    public static String[] customDelimeter(String input) {
+        Matcher m = Pattern.compile("//(.)\n(.*)").matcher(input);
+        if(m.find()) {
+            String customDelimeter = m.group(1);
+            String[] tokens = m.group(2).split(customDelimeter);
+            return tokens;
+        }
+        return input.split(",|:");
+    }
+
+    public static int stringCastingInt(String input) {
+        int i = Integer.parseInt(input);
+        return i;
```

Comment:
아래와 같이 한 줄로 줄여서 작성해 볼 수 있을 것 같습니다.
```suggestion
        return Integer.parseInt(input);
```